quodlibet/quodlibet/qltk/queue.py

```diff
@@ -121,16 +123,27 @@ def __init__(self, library, player):
         outer.pack_start(close_button, False, False, 6)
         self.set_label_fill(True)
 
-        cb = ConfigCheckButton(
-            _("_Random"), "memory", "shufflequeue")
-        cb.connect('toggled', self.__queue_shuffle, self.queue.model)
-        cb.set_active(config.getboolean("memory", "shufflequeue"))
-        cb.set_no_show_all(True)
-        left.pack_start(cb, False, True, 0)
+        rand_checkbox = ConfigCheckButton(
+                _("_Random"), "memory", "shufflequeue")
+        rand_checkbox.connect('toggled',
+                              self.__queue_shuffle,
+                              self.queue.model)
+        rand_checkbox.set_active(config.getboolean("memory", "shufflequeue"))
+        rand_checkbox.set_no_show_all(True)
+        left.pack_start(rand_checkbox, False, True, 0)
+
+        stop_checkbox = ConfigCheckButton(
+            _("Stop After Empty"), "memory", "queue_stop_after_empty")
```

Comment:
I agree. I based it on "Stop After This Song" under Controls, but the wording makes more sense in that case.